docs/基础功能/行内样式绑定状态变量.md

```
@@ -0,0 +1,32 @@
+# 行内绑定状态变量
+
+## 使用说明
+
+设计器提供了一种功能，允许使用定义变量来动态计算组件的位置。这些计算结果随后可以直接应用到组件的行内样式style属性中，从而实现动态样式的调整和布局的优化。
+
+## 操作步骤
+
+1. 添加变量，例如：bgcolor，用来控制背景颜色。
+
+   **图 1**  添加变量
+   ![](./imgs/addVar.png "addVar")
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)